src/filters/index.js

```diff
@@ -130,6 +130,20 @@ exports.key = function (handler, key) {
 // expose keycode hash
 exports.key.keyCodes = keyCodes
 
+exports.debounce = function (handler, delay) {
```

Comment:
This implementation doesn't pass on the arguments the handler gets. In fact you can just use the internal `_.debounce` to wrap the handler.